deploy/operator.yaml

```diff
@@ -1,247 +1,271 @@
 apiVersion: v1
 kind: Namespace
 metadata:
+  labels:
+    app: seccomp-operator
   name: seccomp-operator
 ---
 apiVersion: v1
 kind: ServiceAccount
 metadata:
+  labels:
+    app: seccomp-operator
   name: seccomp-operator
   namespace: seccomp-operator
 ---
 apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRole
 metadata:
+  labels:
+    app: seccomp-operator
   name: config-map-reader
   namespace: seccomp-operator
 rules:
-- apiGroups: [""]
-  resources: ["configmaps"]
-  verbs: ["get", "watch", "list"]
-- apiGroups: [""]
-  resources: ["events"]
-  verbs: ["create", "patch"]
+- apiGroups:
+  - ""
+  resources:
+  - configmaps
+  verbs:
+  - get
+  - watch
+  - list
+- apiGroups:
+  - ""
+  resources:
+  - events
+  verbs:
+  - create
+  - patch
 ---
 apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRoleBinding
 metadata:
+  labels:
+    app: seccomp-operator
   name: config-map-reader-binding
   namespace: seccomp-operator
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: config-map-reader
 subjects:
 - kind: ServiceAccount
   name: seccomp-operator
   namespace: seccomp-operator
-roleRef:
-  kind: ClusterRole
-  name: config-map-reader
-  apiGroup: rbac.authorization.k8s.io
 ---
 apiVersion: v1
+data:
+  nginx-1.19.1.json: '{"defaultAction":"SCMP_ACT_ERRNO","architectures":["SCMP_ARCH_X86_64","SCMP_ARCH_X86","SCMP_ARCH_X32"],"syscalls":[{"names":["accept4","access","arch_prctl","bind","brk","capget","capset","chdir","chown","clone","close","connect","dup2","epoll_create","epoll_ctl","epoll_pwait","epoll_wait","eventfd2","execve","exit","exit_group","faccessat","fadvise64","fchdir","fchown","fcntl","fgetxattr","fsetxattr","fstat","fstatfs","futex","getcwd","getdents","getdents64","getegid","geteuid","getgid","getpid","getppid","getrlimit","getuid","ioctl","io_setup","listen","lseek","mkdir","mmap","mprotect","munmap","nanosleep","newfstatat","open","openat","pipe","prctl","pread64","prlimit64","pwrite64","read","recvfrom","recvmsg","rename","rt_sigaction","rt_sigprocmask","rt_sigreturn","rt_sigsuspend","sched_getaffinity","seccomp","sendfile","sendmsg","setgid","setgroups","setitimer","set_robust_list","setsockopt","set_tid_address","setuid","sigaltstack","socket","socketpair","stat","statfs","sysinfo","umask","uname","unlink","utimensat","wait4","write","writev"],"action":"SCMP_ACT_ALLOW","args":[],"comment":"","includes":{},"excludes":{}}]}'
 kind: ConfigMap
 metadata:
-  name: seccomp-operator-profile
+  annotations:
+    seccomp.security.kubernetes.io/profile: "true"
+  labels:
+    app: seccomp-operator
+  name: default-profiles
   namespace: seccomp-operator
```

Comment:
nitpick: any idea why the `ConfigMap` gets generated with `data` before `kind`? Is it just alphabetical?

Reply:
I think so, not sure if this is configurable by kustomize… :thinking: